File: 34. Find First and Last Position of Element in Sorted Array.py
class Solution:
    def searchRange(self, nums: List[int], target: int) -> List[int]:
        l = []
        s=[]
        nums.sort()
        for i in range(len(nums)):
            if target>=nums[i]:
                if target==nums[i]:
                    l.append(i)
        if len(l)>1:
            s.append(l[0])
            s.append(l[-1])
            return s
        elif len(l)==1:
            l.append(l[0])
            return l
        else:
            return [-1,-1]

File: 34. Find First and Last Position of Element in Sorted Array.cpp
class Solution {
public:
    vector<int> searchRange(vector<int>& nums, int target) {
        vector<int>lst;
        int cnt=-1,i=0;
        while(i<nums.size()){
            if(nums[i]==target){
                cnt=i;
                lst.push_back(cnt);
                int val=i;
                while(val<nums.size()){
                    if(nums[val]==target) cnt=val;i++;val++;
                }
            }
            i++;
        }
        if(cnt>=0) {
            lst.push_back(cnt); 
            return lst;}
        lst.push_back(-1);
        lst.push_back(-1);
        return lst;
    }
};
